chanzuckerberg/single-cell-data-portal
# fix: change the datasets response shape

- #3120

### Reviewers
**Functional:** @danieljhegeman 

**Readability:** @ebezzi 

---


## Changes
- change name to title in dataset
- change is_primary_data to list of boolean in dataset

## QA steps (optional)
- updated tests to reflect changes
- added a test to verify the mapping of is_primary_data is correct.
- added test to verify the mapping of name to title in a dataset is correct.

## Notes for Reviewer
